app/views/checkout/_summary.html.haml

```diff
@@ -1,35 +1,29 @@
 %orderdetails
-  = form_for current_order, url: "#", html: {"ng-submit" => "purchase($event, checkout)"} do |f|
```

Comment:
I guess this is the line that prints a from tag with class edit_order. I couldn't see that tag anywhere else in this diff. Is this what you were looking for?